src/handlers/Ollama/OllamaHandler.py

```diff
@@ -0,0 +1,102 @@
+import os
+import requests
+import json
+import re
+
+from src.models.Diagram import Diagram
+from src.handlers.BaseHandler import BaseHandler
+
+
+class OllamaHandler(BaseHandler):
+
+    def initialize(self):
+        """
+        This method is used to initializes everything the handler needs in order to work.
+
+        For Ollama, this method will load all the ModelFiles defined in the configuration file.
+        """
+
+        reponses = []
+        if "modelFiles" in self.configuration["ai-models"]["ollama"]:
+            for model_file in self.configuration["ai-models"]["ollama"]["modelFiles"]:
```

Comment:
Prefer that each configuration have its own configuration, not all configuration

Reply:
nice idea, done